porch/pkg/engine/engine.go

```diff
@@ -127,7 +131,12 @@ func (p *PackageRevision) KubeObjectName() string {
 }
 
 func (p *PackageRevision) GetResources(ctx context.Context) (*api.PackageRevisionResources, error) {
-	return p.repoPackageRevision.GetResources(ctx)
```

Comment:
Is this change just a leftover from previous drafts of the PR?

Reply:
ugh, yes. I guess it's equivalent

Reply:
I can clean it up though, if you want

Reply:
Not critical, just wanted to make sure there wasn't something here I was missing.